third_party/move/move-compiler/src/parser/syntax.rs

```diff
@@ -3133,7 +3137,7 @@ fn parse_use_alias(context: &mut Context) -> Result<Option<Name>, Box<Diagnostic
 //      Module =
 //          <DocComments> ( "spec" | "module") (<LeadingNameAccess>::)?<ModuleName> "{"
 //              ( <Attributes>
-//                  ( <UseDecl> | <FriendDecl> | <SpecBlock> |
+//                  ( <UseDecl> | <FriendDecl> | <SpecBlock> | <Invariant> |
```

Comment:
@wrwg should this `Invariant` be included in the `SpecBlock`?

Reply:
This is correct, see https://aptos.dev/en/build/smart-contracts/prover/spec-lang#global-invariants.